BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -495,7 +497,17 @@ - (void)prepareRequest:(BOXRequest *)request
         BOXSharedItemRequest *shareItemRequest = (BOXSharedItemRequest *)request;
         shareItemRequest.sharedLinkHeadersHelper = self.sharedLinksHeaderHelper;
     }
-    
+
+    // If the request is currently not set to perform immediately,
+    // Check if we should set it to perform immediately or not
+    if (request.shouldPerformRequestImmediately == NO) {
+       request.shouldPerformRequestImmediately =
+            //should background requests perform immediately
```

Comment:
nit: this comment seem redundant since it reads same as the line of code

Reply:
done